### PR TITLE
Ecto 2.0

### PR DESCRIPTION
**Pull request by vanstee**
Warnings will be removed in a future PR

### CODE REVIEW

File: lib/cog/repository/bundles.ex [hunk 1]
```diff
@@ -464,7 +464,7 @@ defmodule Cog.Repository.Bundles do
     from bv in BundleVersion,
     join: b in assoc(bv, :bundle),
     where: b.name == ^site,
-    where: bv.version == @permanent_site_bundle_version,
+    where: bv.version == ^@permanent_site_bundle_version,
```
Comment (christophermaier) [inline lib/cog/repository/bundles.ex]: Was this actually causing a problem?

Comment (vanstee) [inline lib/cog/repository/bundles.ex]: Yes, it failed to compile.

Comment (christophermaier) [inline lib/cog/repository/bundles.ex]: That's interesting... I'd thought the compiler would see that the same as putting a string literal in there.

In any event, good find 👍

**Comment by christophermaier** [comment]
Good stuff! :shipit:

File: lib/cog/repository/bundles.ex (hunk 1 above)
Comment (vanstee) [inline lib/cog/repository/bundles.ex]: Yep surprised me too.

**Pull request merged**